web/src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import { usePathname } from "next/navigation";
+
+import { Navpill } from "src/components/Navigation/Navpill/Navpill";
+import { Sidebar } from "src/components/Navigation/Sidebar/Sidebar";
+
+import { css } from "@/styled-system/css";
+
+import { SIDEBAR_WIDTH } from "./useNavigation";
+
+const ROUTES_WITHOUT_NAVPILL = ["/new"];
+
+const isNavpillShown = (path: string | null) =>
+  ROUTES_WITHOUT_NAVPILL.includes(path ?? "");
+
+export function Navigation() {
+  const pathname = usePathname();
+
+  return (
+    <>
+      {/* MOBILE */}
+      <div
+        id="mobile-nav-container"
+        className={css({
+          display: {
+            base: isNavpillShown(pathname) ? "none" : "unset",
+            md: "none",
+          },
+        })}
+      >
+        <Navpill />
+      </div>
+
+      {/* DESKTOP */}
+      <div
+        id="desktop-nav-container"
+        className={css({
+          display: {
+            base: "none",
+            md: "flex",
+          },
+          minWidth: SIDEBAR_WIDTH,
```

Comment:
The other usage: https://github.com/Southclaws/storyden/pull/30/files#r1311796602
The explanation why this style rule exists in two places:

Reply:
Okay I went for a walk and realised why I am misunderstanding this! And I'm so silly...

Of course the variable cannot be accessed because the CSS is generated ahead of time and the variable itself exists within the JS world not the CSS world. Obviously it needs to be static and literal at build time.